UI/src/app/lp-header/lp-header.component.html

```diff
@@ -2,7 +2,13 @@
     <h1 class="text-3xl text-yellow-600 font-semibold">LitPaths</h1>
 
     <div class="flex items-start">
-        <lp-feedback-button></lp-feedback-button>
+        <button aria-label="Give feedback about LitPaths"
```

Comment:
You don't need to complicate your markup like this, aim for making it as simple as possible.

`<a href=".." target="_blank"><span class="sr-only">Some description</span><i class="icon-mail"></i></a>`

The `.sr-only` is a [tailwind class](https://tailwindcss.com/docs/screen-readers#class-reference) for accessibility.